src/components/ContactsList/ContactsList.jsx

```diff
@@ -1,22 +1,59 @@
+/* global cozy */
 import React from "react";
 import PropTypes from "prop-types";
 import { sortGivenNameFirst, getDisplayedName } from "./";
 import ContactRow from "./ContactRow";
+import { Button, IntentOpener } from "cozy-ui/react";
 
 const ContactHeaderRow = props => <div className="divider">{props.header}</div>;
 ContactHeaderRow.propTypes = {
   header: PropTypes.string.isRequired
 };
 
-const ContactsEmptyList = () => (
-  <div>
-    <p>No Contact</p>
-    <p>Try to create one with the big button on the top</p>
-    <p>
-      Or to import your contacts with this <a href="#">link</a>
-    </p>
-  </div>
-);
+class ContactsEmptyList extends React.Component {
+  state = {
+    hasConnector: false
+  };
+
+  componentWillMount() {
+    // cozy-client-js is needed for intents
+    const root = document.querySelector("[role=application]");
+    const data = root.dataset;
+    cozy.client.init({
+      cozyURL: `${window.location.protocol}//${data.cozyDomain}`,
+      token: data.cozyToken
+    });
+  }
+
+  afterConnection = () => {
+    this.setState({ hasConnector: true });
+  };
+
+  render() {
+    const { hasConnector } = this.state;
+    const { t } = this.context;
+
+    return (
+      <div className="list-empty">
+        <h1>{t("empty.title")}</h1>
+        <p>{t("empty.description")}</p>
+
+        {hasConnector ? (
+          <p>{t("empty.after")}</p>
+        ) : (
+          <IntentOpener
+            action="CREATE"
+            doctype="io.cozy.accounts"
+            options={{ slug: "google" }}
+            onComplete={this.afterConnection}
```

Comment:
if we dismis the intent modal, the state is not correct.

Reply:
Uhm no, what do you mean? If we dismiss the modal, then nothing changes and the button is still there. Is something different hapenning on your end?

Reply:
Found, and fixed after further discussions!

Reply:
If you don't "connect" the konnector, the state is "please wait for your contacts" instead of the state "please use the konnetor to upload your contacts".

But with the last commit, that checks if the parameter is `null`, it works like a charm. So, I can approve this PR.